Jenkinsfile

```diff
@@ -205,7 +205,7 @@ pipeline {
                 parent_image.inside() {
                   unstash(name: srcStash(params.release_label))
                   unstash(name: recipeStash(recipe_label))
-                  sh "generate_bundle_templates --src-dir $src_dir --template-dir $debian_dir --recipe $recipe_path"
+                  sh "ROS_PYTHON_VERSION=2 generate_bundle_templates --src-dir $src_dir --template-dir $debian_dir --recipe $recipe_path"
```

Comment:
TODO for ROSCon: figure out where to configure stuff like this. Anything that could be in a package.xml conditional should really be available in all tailor-distro docker env (parent and bundle-specific dockers).

Maybe we can just have a `bundle_environment` dict like we have for ansible deployments?